app/jobs/autoscaling.py

```diff
@@ -208,15 +188,24 @@ class ASG(object):
     job_dispatch_lag_grace_period_seconds = 60
     job_tag_keep_alive_grace_period_seconds = 60
 
-    def __init__(self, service, environment, asg_list, tag_list, draining_tag, job_tag_prefix, max_job_dispatch_lag_seconds, job_tag_keep_alive_seconds):
+    numa_partitions = 2 # i3.16xlarge or i3.metal
+    desired_queue_depth = 3 # initialization takes time: it's fine if some number of chunks have to run sequentially rather than in parallel
+    instance_limit = 24 # this autoscaler will never go beyond this number unless operator intervenes
```

Comment:
Isn't this what the "max_capacity" setting on the ASG is for?

Reply:
true